include/rapid_pbd/baxter_actions.h

```diff
@@ -0,0 +1,71 @@
+// This file adapts the BAXTER actuator actions, which use a BAXTER-specific
+// interface, to use the more standard interfaces in the control_msgs package.
+
+#ifndef _RAPID_PBD_BAXTER_H_
+#define _RAPID_PBD_BAXTER_H_
+
+#include <string>
+
+#include "actionlib/client/simple_action_client.h"
+#include "actionlib/server/simple_action_server.h"
+#include "control_msgs/FollowJointTrajectoryAction.h"
+#include "control_msgs/GripperCommandAction.h"
+#include "control_msgs/SingleJointPositionAction.h"
+
+#include "rapid_pbd_msgs/ArmControllerState.h"
```

Comment:
Delete the includes for ArmControllerState, FreezeArm, and RelaxArm, since they are unused.